scripts/do-release.sh

```diff
@@ -101,17 +102,12 @@ function update_go_mod() {
 }
 
 function update_dependencies() {
-    local msg="$1"
-    shift
-
     clone_and_create_branch "update-dependencies-${release['branch']:-devel}"
 
-    for dependency; do
-        update_go_mod "$dependency"
-    done
+    [ ! -f "projects/${project}/go.mod" ] || update_go_mod
```

Comment:
```suggestion
    update_go_mod
```

Should just call it directly, it already handles this case and in case the go.mod is only inside a sub-directory it will handle it correctly, while the check on this line will fail to up[date that file.

Reply:
But `update_go_mod` returns 1 if there is no `go.mod` file, wouldn’t that fail the release?

Reply:
Yes hence I suggested to inline this function here and handle it here, otherwise that code in `update_go_mod` is useless with this check